### DISCUSSION TITLE
GemRB and Flare

### DISCUSSION THREAD

**Discussion by WithinAmnesia** [flareteam/flare-game#948] [Q&A]
https://gemrb.org/ + https://github.com/gemrb/gemrb + https://github.com/gemrb/gemrb/blob/master/CONTRIBUTING.md + https://forums.beamdog.com/discussion/80853/modifying-the-gemrb-demo + https://www.youtube.com/watch?v=Qy1OxwwTbbM&ab_channel=TuDawid + https://www.youtube.com/watch?v=Ybif2SgGMdA&ab_channel=DebConfVideos
GemRB and Flare are already related and use common assets and are both commercially viable open source classic 1990's grand RPG engines hosted on Github made in C++. I wonder what can be combined and what are peoples thoughts? I have just been wondering about all of this and figured to think ponder and wonder openly with the community.

**Comment by basxto** [comment]
Where do they use common assets?

**Reply by WithinAmnesia** [reply]
The sound effects that are open source and art assets and also the general community spirit and being helpful to everyone. Mostly the sound and art assets. Maybe some ui or down-scaled pixelated art as well. A lot of common themes are shared throughout too and common ways to work on the games arise.

**Reply by dorkster** [reply]
To clarify: GemRB **doesn't** ship with any flare-game assets. Which is what I think @basxto was asking.

I suppose someone *could* use those assets to make an Infinity Engine game that works with GemRB. But no one's done that to my knowledge.

**Reply by basxto** [reply]
It’s not wrong. But it’s really only a two map demo so it can be run and tested without needing the proprietary games files. I don’t see any focus on creating free content whatsoever.

Both use https://opengameart.org/content/rpg-gui-construction-kit-v10, which is the most noticeable. They just happen to source from the same third party sites. But I doubt that the engines could create much compatibility between them since GemRB has it’s focus on being compatible with the games it’s engine it recreates.

Relevant GemRB links would be:

* https://github.com/gemrb/gemrb/tree/master/demo
* https://github.com/gemrb/gemrb-assets

**Reply by basxto** [reply]
At second sight … GemRB looks a lot more flexible than flare. It might be possible to handle flare mods with plugins and scripts on GemRBs side.

https://gemrb.org/Engine-overview.html#plugins-gemrbplugins
